### ISSUE TITLE
Cab we enable CORS for webfonts.css?

### ISSUE THREAD

**Issue by ragulka** [feat-agency/vite-plugin-webfont-dl#50]
I ran into an issue where I need to fetch the `webfonts.css` using axios, but it looks like the plugin does not add any CORS headers, resulting in `Access to XMLHttpRequest at 'https://site.test:5173/webfonts.css' from origin 'https://site.test' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.` error.

Interestingly, when serving font files, this plugin [_does_ add the `Access-Control-Allow-Origin: *` header](https://github.com/feat-agency/vite-plugin-webfont-dl/blob/f78db49f96d4b997c9913ea88492729e6543d103/src/index.ts#L338), but [not when serving the CSS file](https://github.com/feat-agency/vite-plugin-webfont-dl/blob/f78db49f96d4b997c9913ea88492729e6543d103/src/index.ts#L314-L326).

Is this for a reason, or would it be possible to add the same header when serving the CSS file?

FWIW, here's the confioguration I use:

```js
webfontDownload([
    'https://fonts.bunny.net/css2?family=Lato:wght@400;600;700&display=swap',
    'https://fonts.googleapis.com/css2?family=Bitter:ital,wght@0,100..900;1,100..900&display=swap',
    'https://fonts.googleapis.com/css2?family=Cormorant+Unicase:wght@300;400;500;600;700&display=swap',
])
```

I tried using the following options, but these had no effect on this issue:

```js
{
    injectAsStyleTag: false,
    embedFonts: true,
}
```

**Comment by 0xb4lint** [comment]
Thanks for raising the issue @ragulka!

Just released `v3.9.2` with CSS CORS support.

**Comment by ragulka** [comment]
> Just released `v3.9.2` with CSS CORS support.

@0xb4lint this is awesome, works just as expected - thanks :)
